software/due-date-changes.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import re
+import sys
+import time
+
+from datetime import datetime
+from jira import JIRA
+
+
+def dump(obj):
+    for attr in dir(obj):
+        print("obj.%s = %r" % (attr, getattr(obj, attr)))
+
+
+def print_help(stream=sys.stderr):
+    help_msg = """Usage: {0} [--projects] LIST,OF,PROJECTS
+"""
+    stream.write(help_msg.format(sys.argv[0]))
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-projects", 
+        help="Comma-separated list of project names (ie. HTCONDOR,SOFTWARE)",
+        default="HTCONDOR,SOFTWARE")
+    parser.add_argument("-o", 
+        help="Output file",
+        default=None)
+    args = parser.parse_args()
+
+    projects = args.projects
+    output_file = args.o
+
+    return {
+        "projects": projects,
+        "output_file": output_file
+    }
```

Comment:
Up to you, but feel free to collapse these here, i think without any loss of clarity

```suggestion
    return {
        "projects": args.projects,
        "output_file": args.o
    }
```